packages/manager/modules/vps/src/veeam/modal/schedule-backup/vps-schedule-backup.html

```diff
@@ -0,0 +1,26 @@
+<oui-modal
+    data-primary-label="{{ ::'vps_configuration_veeam_schedule_backup_action_confirm' | translate }}"
+    data-primary-action="$ctrl.onRescheduledAutomatedBackupConfirmed()"
+    data-primary-disabled="!$ctrl.model.rescheduleTimepicker"
+    data-secondary-action="$ctrl.onRescheduledAutomatedBackupCanceled()"
+    data-secondary-label="{{ ::'vps_configuration_veeam_schedule_backup_action_cancel' | translate }}"
+    data-on-dismiss="$ctrl.onModalCanceled()"
+    data-heading="{{:: 'vps_configuration_veeam_schedule_backup_title' | translate}}"
+    data-loading="$ctrl.isLoading"
+>
+    <p
+        class="oui-modal__text"
+        data-translate="vps_configuration_veeam_schedule_backup_description"
+    ></p>
+
+    <p class="mt-3 mb-3">
```

Comment:
FYI existing following shorthand notation:

```suggestion
    <p class="my-3">
```

see: https://getbootstrap.com/docs/4.0/utilities/spacing/#notation

**alternative**: you can simply apply a `mb-x` to the sibling `<p>` element (L11).

Reply:
I searched for something like this shorthand but i don't found it. Thanks.
Changed.